modules/analyze-images-and-deployments-without-observed-cves.adoc

```diff
@@ -2,18 +2,109 @@
+An image that actually contains vulnerabilities can appear in this list inadvertently. For example, if Scanner was able to scan the image and it is known to {rh-rhacs-first}, but the scan was not successfully completed, {product-title-short} cannot detect vulnerabilities.
 
 This scenario occurs if an image has an operating system that {product-title-short} Scanner does not support. {product-title-short} displays scan errors when you hover over an image in the image list or click the image name for more information.
-====
\ No newline at end of file
+====
+
+.Procedure
+
+. In the {product-title-short} portal, go to *Vulnerability Management* -> *Results*.
+. Click *More Views* and select *Images without CVEs*.
+. To filter the list of results by entity, for example, to search for a specific named CVE, select the appropriate filters and attributes.
++
+To select multiple entities and attributes, click the right arrow icon to add another criteria. Depending on your choices, enter the appropriate information such as text, or select a date or object.
++
+The filter entities and attributes are listed in the following table.
++
+[NOTE]
+====
+The *Filtered view* icon indicates that the displayed results were filtered based on the criteria that you selected. You can click *Clear filters* to remove all filters, or remove individual filters by clicking on them.
+====
++
+.CVE filtering
+[cols="2",options="header"]
+|===
+|Entity|Attributes
+
+|Image
+a|
+* *Name*: The name of the image.
+* *Operating system*: The operating system of the image.
+* *Tag*: The tag for the image.
+* *Label*: The label for the image.
+* *Registry*: The registry where the image is located.
+|CVE
+a|
+* *Name*: The name of the CVE.
+* *Discovered time*: The date when {product-title-short} discovered the CVE.
+* *CVSS*: The severity level for the CVE.
++
+The following values are associated with the severity level for the CVE:
++
+** *is greater than*
+** *is greater than or equal to*
+** *is equal to*
+** *is less than or equal to*
+** *is less than*
+* *EPSS probability*: The likelihood that the vulnerability will be exploited according to the link:https://www.first.org/epss/model[Exploit Prediction Scoring System (EPSS)]. This EPSS data provides a percentage estimate of the probability that exploitation of this vulnerability will be observed in the next 30 days. The EPSS collects data of observed exploitation activity from partners, and exploitation activity does not mean that an attempted exploitation was successful. The EPSS score should be used as a single data point _along with other information_, such as the age of the CVE, to help you prioritize the vulnerabilities to address. For more information, see link:https://access.redhat.com/articles/7106599[{product-title-short} and EPSS].
```

Comment:
In a quick check of staging demo last week, it looks like CVEs table will be empty.

Although search filter probably has **EPSS probability** because code is reused, it is worthwhile to consider deleting CVE filtering from this doc.

Reply:
Oops, thank you for catching this - I think it's a cut and paste error.